cmd/frontend/main.go

```diff
@@ -153,7 +153,7 @@ func main() {
 			//nolint:errcheck
```

Comment:
Is there a plan to remove these?

Reply:
This is a weird one. The logger can technically return an error, which is not being checked. Whether that's a useful error check is debatable. So, we have a few options:

1. Check for errors on logging, when using this logger.
2. Leave the `//nolint` annotations in place.
3. Add an exception to recognize this pattern of error and ignore it.
4. Use a different logging library that doesn't return errors.

Not sure which I would prefer yet, personally.